packages/web/src/shared/components/mobileNavbar/index.tsx

```diff
@@ -0,0 +1,64 @@
+import clsx from 'clsx'
+import Link from 'next/link'
+import { usePathname } from 'next/navigation'
+
+import { IoInvertMode } from 'react-icons/io5'
+
+import { LogoNouns } from '@/shared/icons/LogoNouns'
+
+import { Button } from '../button'
+
+interface Props {
+  isNavbarOpened: boolean
+  routes: Route[]
+}
+
+interface Route {
+  name: string
+  route: string
+}
+
+const noundAndDarkModeButtons = 'bg-black-lighter hover:bg-black-light cursor-pointer rounded p-1'
+
+export function MobileNavbar({ isNavbarOpened, routes }: Props) {
+  const pathname = usePathname()
+
+  if (!isNavbarOpened) return null
```

Comment:
brisa: eu já li esse trecho vezes o suficiente pra questionar se isso é uma boa, pessoalmente eu consigo me convencer facilmente que seria melhor fazer um "toggle" usando CSS (usando `clsx` e o `hidden`), pois isso evitaria fazer o React montar e desmontar toda essa árvore de elementos toda vez que a `navbar` for aberta.

Não acho que tem algo especificamente errado com esse `if` (apenas acho feio pra porra) mas fico extremamente curioso a respeito de performance (que também dá pra argumentar que nesse caso não vai ser muito significativo e lá lá lá, mas eu estou aqui pra ser chato)